### PR TITLE
Fix retries for appix

### PR DESCRIPTION
**Pull request by jackTheRipper**
** Do **

Fix the retry logic for `appix submit`

### CODE REVIEW

File: appcatalog/submitToCatalog.go [hunk 1]
```diff
@@ -96,7 +95,7 @@ func doSubmit(req *http.Request, verbose bool) (acceptanceQueryURL string, err e
 	}
 
 	acceptanceQueryURLPath, _ := responseObject.Links["acc:query"]
-
+	res.Body.Close()
```
Comment (markvincze) [inline appcatalog/submitToCatalog.go]: Why aren't we `defer`ing this above?

**Comment by alexmiranda** [comment]
Please make sure the same thing is done for `push`.

File: appcatalog/submitToCatalog.go (hunk 1 above)
Comment (jackTheRipper) [inline appcatalog/submitToCatalog.go]: You are right I wasn't thinking about it. Fixed.

**Comment by jackTheRipper** [comment]
@alexmiranda : fixed

**Pull request merged**